funding/commitment_type_negotiation.go

```diff
@@ -307,6 +307,73 @@ func explicitNegotiateCommitmentType(channelType lnwire.ChannelType, local,
 
 		return lnwallet.CommitmentTypeSimpleTaproot, nil
 
+	// Simple taproot channels overlay only.
+	case channelFeatures.OnlyContains(
+		lnwire.SimpleTaprootOverlayChansOptional,
+	):
+
+		if !hasFeatures(
+			local, remote,
+			lnwire.SimpleTaprootOverlayChansOptional,
+		) {
+
+			return 0, errUnsupportedChannelType
+		}
+
+		return lnwallet.CommitmentTypeSimpleTaproot, nil
+
+	// Simple taproot overlay channels with scid only.
+	case channelFeatures.OnlyContains(
+		lnwire.SimpleTaprootOverlayChansOptional,
+		lnwire.ScidAliasRequired,
+	):
+
+		if !hasFeatures(
+			local, remote,
+			lnwire.SimpleTaprootOverlayChansOptional,
+			lnwire.ScidAliasOptional,
+		) {
+
+			return 0, errUnsupportedChannelType
+		}
+
+		return lnwallet.CommitmentTypeSimpleTaproot, nil
+
+	// Simple taproot overlay channels with zero conf only.
+	case channelFeatures.OnlyContains(
+		lnwire.SimpleTaprootOverlayChansOptional,
+		lnwire.ZeroConfRequired,
+	):
+
+		if !hasFeatures(
+			local, remote,
+			lnwire.SimpleTaprootOverlayChansOptional,
+			lnwire.ZeroConfOptional,
+		) {
+
+			return 0, errUnsupportedChannelType
+		}
+
+		return lnwallet.CommitmentTypeSimpleTaproot, nil
+
+	// Simple taproot overlay channels with scid and zero conf.
+	case channelFeatures.OnlyContains(
+		lnwire.SimpleTaprootOverlayChansOptional,
+		lnwire.ZeroConfRequired,
+		lnwire.ScidAliasRequired,
+	):
+
+		if !hasFeatures(
+			local, remote,
+			lnwire.SimpleTaprootOverlayChansOptional,
+			lnwire.ZeroConfOptional,
```

Comment:
Not sure I grok the pattern here, but shouldn't this also include `lnwire.ScidAliasOptional`?

Reply:
Was copy pasted from above, but I think it's actually a lingering bug. Will look into it.